xamarin/Xamarin.Forms
# Limit scrollbar application to ItemsViewRenderer

### Description of Change ###

Fixes the _"scrollbar flash"_ issue on CollectionView.

This change applies the scrollbar style directly to the ItemsViewRenderer and _only_ to the ItemsViewRenderer; other child controls are left unchanged.

### Issues Resolved ### 

- fixes #8626
- fixes #10497
- fixes #7281
- fixes #8361
- fixes #10294
- fixes #9947

### API Changes ###

None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Launch Control Gallery and navigate to the Issue 10497. If loading the page you don't see the scrollbar in each CollectionView Item, the test has passed.

### PR Checklist ###
